src/Schema/Values/FilterParams.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Master list of filter params available in web and iOS
+ *
+ * Intended action by a user that triggered an event
+ * @packageDocumentation
+ */
+export enum FilterParams {
```

Comment:
Here, I want to have all the filters that can possibly change (like `dimensionRange`, `priceRange` etc.) but I am wondering if I also need to have some list of value that those filters can be.

Reply:
`FilterParams` should be an `interface` and define the types of content that are allowed, for example `string` or `boolean`.

We can also add an enum for values such as `medium` which would provide a master list of filer options for that key.